### PR TITLE
Updated Member list

### PR DESCRIPTION
**Pull request by ujdcodr**

### CODE REVIEW

**Comment by ujdcodr** [comment]
please review

**Comment by dip-kush** [comment]
@kavya-a @cjamadagni Please Review it

File: index.html [hunk 1]
```diff
@@ -291,7 +291,7 @@ <h5 class="light grey-text text-lighten-3"></h5>
 										<li>Ajin Tom</li>
 										<li>Akankshya Kar</li>
 										<li>Akshay Sangat</li>
-										<li><a href="https://sigsevneo.github.io">Anagh Singh</a></li>
```
Comment (kavya-a) [inline index.html]: leave as it was

**Comment by ujdcodr** [comment]
Chirag said remove the link

**Comment by ujdcodr** [comment]
Why was the issue removed?

**Comment by ujdcodr** [comment]
I have tested the website locally....it works

**Pull request merged**